test/functional/content_view/version/republish_repositories_test.rb

```diff
@@ -26,7 +26,7 @@ module HammerCLIKatello
       expect_foreman_task('3')
 
       result = run_cmd(%w(content-view version republish-repositories --organization org1
-                          --content-view cv --version 1))
+                          --content-view cv --version 1 --force true))
```

Comment:
Isn't it dangerous that few checks wont apply? I don't know what this flag is for exactly, but doesn't it mean that if this flag is set then some logic in the command is dropped?

Reply:
Sadly, this command now won't work unless --force is passed in. When I try it without it, it companies it needs and when i set it to false, it says it must be true. This is the pr that introduced it.

https://github.com/Katello/katello/commit/79e606ac3ffcb416c3c03baeca12e083e24bbe84

Reply:
Ah, right. Well, if it *is* required in the API in such way, then it's OK, my bad :/